docs/content/concepts/metadata-tags.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Metadata & tags | Dagster Docs"
+description: "Dagster offers several ways to provide useful information and documentation alongside your data pipelines, including metadata and tagging."
+---
+
+# Metadata & tags
+
+One of the core benefits of Dagster is fostering collaboration between the engineers who build data pipelines and the end users who consume the data pipelines produce.
+
+Dagster offers several ways to provide useful information and documentation alongside your data pipelines, including metadata and tagging.
+
+---
+
+## Benefits
+
+Using metadata and tags helps you:
+
+- Create built-in documentation that makes your pipelines easy to understand
+- Provide useful context for other users of your project (and your future self!)
+- Improve the ease of debugging when issues arise
+- Organize definitions in your Dagster project and improve filtering in the Dagster UI
+
+---
+
+## How metadata works
+
+There are two main types of metadata in Dagster: definition and runtime. In code, you can attach metadata to definitions like assets, jobs, and runs. You'll then be able to view the metadata in the Dagster UI.
+
+### Definition metadata
+
+**Definition metadata** is information that's fixed or doesn't frequently change. For example, definition metadata could be:
+
+- A description of an asset
+- [Adding an asset to a group](/concepts/assets/software-defined-assets#grouping-assets)
+- Links to resources that are [attached to a job](/concepts/metadata-tags/op-job-metadata)
+
+Refer to the [Attaching definition metadata](/concepts/assets/software-defined-assets#attaching-definition-metadata) section of the [Software-defined Assets](/concepts/assets/software-defined-assets) documentation for more information.
+
+### Runtime metadata
+
+**Runtime** metadata is information that changes after an action occurs. This could be the date an asset was materialized, how many records were processed, how long an asset took to materialize, etc. Refer to the [Recording materialization metadata](/concepts/assets/software-defined-assets#recording-materialization-metadata) section of the [Software-defined Assets](/concepts/assets/software-defined-assets) documentation for more information.
```

Comment:
Similar to my comment above, "the date an asset was materialized" is metadata in a broad sense, but not in the strict sense that I think we should be using the term within this doc.  That's because we automatically store it as one of the core fields on an asset materialization, rather than expecting users to set it as metadata.  The other two examples in this list are solid though.